step-plans/step-plans-base-artefacts/src/main/java/step/artefacts/handlers/CheckHandler.java

```diff
@@ -39,6 +39,7 @@ protected void execute_(CheckReportNode node, Check testArtefact) {
 				node.setStatus(ReportNodeStatus.PASSED);
 			} else {
 				node.setStatus(ReportNodeStatus.FAILED);
+				node.addError("The expression '" + testArtefact.getExpression().getExpression() + "' returned false");

```

Comment:
Should we use a specific error code for Assert

Reply:
@jeromecomte as far I can see all error generated by our handlers still create error with default error code 0 , like for Check, PerformanceAssert... wouldn't it make sense to already have dedicated one?